apps/condo/domains/common/hooks/useQueryMappers.ts

```diff
@@ -0,0 +1,46 @@
+import { QueryMeta, SorterColumn, convertSortersToSortBy } from '../utils/tables.utils'
+import { useMemo } from 'react'
+import get from 'lodash/get'
+
+export const useQueryMappers = (queryMetas: Array<QueryMeta>, sortableColumns: Array<string>) => {
+    return useMemo(() => {
+        const validSorts = sortableColumns.reduce((acc, cur) => {
+            return [...acc, `${cur}_ASC`, `${cur}_DESC`]
+        }, [])
+        const validMetas = queryMetas
+            .filter((meta) => meta && meta.keyword && meta.filters && meta.filters.length > 0)
+            .map((meta) => ({ [meta.keyword]: meta }))
+
+        const filtersMap = Object.assign({}, ...validMetas)
+
+        const filtersToWhere = (queryFilters) => {
+            const whereQueries = []
+            for (const [key, value] of Object.entries(queryFilters)) {
```

Comment:
add readable names

Reply:
added and styled this function to look more linear